xnat-web.pkr.hcl

```diff
@@ -86,6 +87,7 @@ build {
       "unzip -o -d $${CATALINA_HOME}/webapps/ROOT /tmp/xnat-web-*.war",
       "sed -i 's/ch.qos.logback.core.rolling.RollingFileAppender/ch.qos.logback.core.ConsoleAppender/' $${CATALINA_HOME}/webapps/ROOT/WEB-INF/classes/logback.xml",
       "cp /tmp/packer_files/setenv.sh $${CATALINA_HOME}/bin/setenv.sh && chmod 0555 $${CATALINA_HOME}/bin/setenv.sh",
+      "cp /tmp/packer_files/mq.png $${CATALINA_HOME}/webapps/ROOT/images/mq.png && chmod 0644 $${CATALINA_HOME}/webapps/ROOT/images/mq.png",
```

Comment:
Is this to copy your custom logo?

Reply:
Whoops! I have removed that line - sorry.